cpanfile.snapshot

```diff
@@ -22,83 +22,83 @@ DISTRIBUTIONS
       Algorithm::Diff::_impl 1.201
     requirements:
       ExtUtils::MakeMaker 0
-  Alien-Build-2.41
```

Comment:
Is this file equivalent to a `cargo.lock`/`package-lock.json`/`requirements.txt`?
If so, would it be possible to split changing dependency versions to a different patch?

Reply:
It is like a requirements.txt. The snapshot file is the list of the actual versions that were installed when all of the latest packages were pulled down. I will see if I can minimize the amount of changes.